test/test_embertemplatelint_executable_detection.vader

```diff
@@ -0,0 +1,22 @@
+Before:
+  call ale#test#SetDirectory('/testplugin/test')
+
+  runtime ale_linters/handlebars/embertemplatelint.vim
+
+After:
+  call ale#test#RestoreDirectory()
+  call ale#linter#Reset()
+
+Execute(ember-template-lint executables runs the right command):
+  call ale#test#SetFilename('ember-template-lint-test-files/app/template.hbs')
+
+  AssertEqual
+  \ ale_linters#handlebars#embertemplatelint#GetCommand(bufnr(''), [2, 0, 0]),
+  \ '%e --json --filename /testplugin/test/ember-template-lint-test-files/app/template.hbs'
```

Comment:
This fails on Windows: https://ci.appveyor.com/project/dense-analysis/ale/builds/32131713?fullLog=true#L316

Reply:
Something like:
```vim
  if has('win32')
    let g:ember_template_lint_command =
    \ '%e --json --filename C:\testplugin\test\ember-template-lint-test-files\app\template.hbs'
  else
    let g:ember_template_lint_command =
    \ '%e --json --filename /testplugin/test/ember-template-lint-test-files/app/template.hbs'
  endif

  AssertEqual
  \ ale_linters#handlebars#embertemplatelint#GetCommand(bufnr(''), [2, 0, 0]),
  \ g:ember_template_lint_command
```
should work.